libraries/AP_Winch/AP_Winch_Daiwa.cpp

```diff
@@ -0,0 +1,227 @@
+#include <AP_Winch/AP_Winch_Daiwa.h>
+#include <GCS_MAVLink/GCS.h>
+
+extern const AP_HAL::HAL& hal;
+
+// true if winch is healthy
+bool AP_Winch_Daiwa::healthy() const
+{
+    // healthy if we have received data within 3 seconds
+    return (AP_HAL::millis() - data_update_ms < 3000);
+}
+
+void AP_Winch_Daiwa::init()
+{
+    // initialise rc input and output
+    init_input_and_output();
```

Comment:
Should just call init() on the superclass and have it do whatever it needs to do.

Reply:
ok, moved init_input-and_output to super class's init, thanks!